utils/chat/chatCrypto.ts

```diff
@@ -0,0 +1,32 @@
+import AES from 'crypto-js/aes'
+import Base64 from 'crypto-js/enc-base64'
+import Utf8 from 'crypto-js/enc-utf8'
+import WordArray from 'crypto-js/lib-typedarrays'
+
+interface encryptedTextWithIv {
```

Comment:
```suggestion
interface encryptedTextWithIV {
```
pour l'acronyme IV par convention (Initialization Vector)